virttest/vt_resmgr/resources/network/tap/tap_network.py

```diff
@@ -0,0 +1,32 @@
+import logging
+
+
+from ...pool import _ResourcePool
+from .tap_port import get_port_resource_class
+
+
+LOG = logging.getLogger("avocado." + __name__)
+
+
+class _LinuxBridgeNetwork(_ResourcePool):
+    _POOL_TYPE = "bridge"
```

Comment:
According to meet_resource_request() called by select_pool(), 
:param resource_type: The resource's type, supported:
                              "volume", "port"